src/views/portfolio/projects/ProjectList.vue

```diff
@@ -173,6 +234,9 @@
         ],
         data: [],
         options: {
+          idField: 'id',
+          parentIdField: 'pid',
```

Comment:
Did some testing and it seems like this does not have to be a numeric ID. We may just as well use the UUIDs, which IMO is preferable. The DT API does not expose technical IDs anywhere else, and I'd prefer if it stays that way.

```suggestion
          idField: 'uuid',
          parentIdField: 'parentUuid',
```

Reply:
I initially tried to use UUIDs to display the child-parent-relationship, however doing so led to some strange behavior, which did not occur if a numeric ID is used.

I've just tried to use UUIDs again, and still get the same bug during testing.
If you look at the attached screenshot, some projects get duplicate entries when expanded and mimic the behavior of their identical counterpart. This only happens when using UUIDs, numerical IDs work just fine, but I cannot explain why or how it happens 😅 

**Behavior with UUIDs:**
![image](https://user-images.githubusercontent.com/113189967/203027619-7853a9db-3eef-4adf-868d-130f8e52cc85.png)

**Behavior with numerical IDs:**
![image](https://user-images.githubusercontent.com/113189967/203031128-be9aa822-75f8-4b84-8869-72ab9de053ec.png)

Reply:
Ah I see, that's unfortunate of course. Alternatively, how about generating a numeric checksum from the UUID, similar to Java's `hashCode` or Murmur2 and use that instead? If the issue is about numbers vs. strings, then this could help.

Reply:
Using the `hashCode` of the UUIDs as values for `id` and `pid` seems to work well 👍
Would you still keep the names of the `JsonProperties` as `id` and `pid`?

Reply:
I think the hashing should be done on the client side, not in the API server. So the `id` and `pid` fields should be removed from the API response. After all this is a workaround for a limitation in the client.